configure.ac

```diff
@@ -1603,45 +1603,21 @@ if test "x$with_libhal" = "xyes"; then
 	fi
 fi
 
-# --with-libpthread {{{
-AC_ARG_WITH(libpthread, [AS_HELP_STRING([--with-libpthread=@<:@=PREFIX@:>@], [Path to libpthread.])],
-[	if test "x$withval" != "xno" \
-		&& test "x$withval" != "xyes"
-	then
-		LDFLAGS="$LDFLAGS -L$withval/lib"
-		CPPFLAGS="$CPPFLAGS -I$withval/include"
-		with_libpthread="yes"
-	else
-		if test "x$withval" = "xno"
-		then
-			with_libpthread="no (disabled)"
-		fi
-	fi
-], [with_libpthread="yes"])
-
 AC_SUBST([PTHREAD_LIBS])
```

Comment:
Nit: While it works either way, I think it's cleaner to have AC_SUBST() after defining and populating the underlying variable.